pwa-devdocs/src/_drafts/venia-pwa-concept/component/purchase-details-page/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: PurchaseDetailsPage component
+---
+
+| Description                | Reference image                               |
+| -------------------------- | :-------------------------------------------: |
+| Purchase details page      | ![Purchase details page]{:width="300px"}           |
+| Order details block        | ![Order details block]{:width="300px"}                     |
+| Order items list           | ![Order items list]{:width="300px"} |
+| Order item                 | ![Order item]{:width="300px"} |
+| Button group               | ![Button group]{:width="300px"} |
+{:style="table-layout:auto"}
+
+[Purchase details page]: {{site.baseurl}}{% link venia-pwa-concept/images/purchase-details-page.png %}
+[Order details block]: {{site.baseurl}}{% link venia-pwa-concept/images/purchase-details-order-details-block.png %}
+[Order items list]: {{site.baseurl}}{% link venia-pwa-concept/images/purchase-details-order-items-list.png %}
+[Order item]: {{site.baseurl}}{% link venia-pwa-concept/images/purchase-details-order-item.png %}
+[Button group]: {{site.baseurl}}{% link venia-pwa-concept/images/purchase-details-button-group.png %}
+
+## Interactions
+
+* **Tap BuyAgain** - Adds item to the cart.
```

Comment:
We have more interactions. We have also share button at least. They should be described here.